README.md

```diff
@@ -30,10 +31,10 @@ Read our Quick Start guide [here](https://unitystation.github.io/unitystation/co
 6. Submit a pull request
 
 > **Warning**
-> The Unitystation editor project greatly prefers a SSD, so it is recommended to have an SSD when developing
+> The Unitystation project **requires** an SSD to run in the unity editor.
```

Comment:
like I said before it doesn't require an SSD , just for the best dev experience in SSD is recommended, 

require = it does not physically run if you don't have it
highly recommended is better for this scenario

Reply:
I think I agree

Reply:
if someone complains again about the editor being unusable while having an HDD I'm changing this back